File: abc/abc346/abc346_b/abc346_b.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int w, b;
  cin >> w >> b;

  string s = "wbwbwwbwbwbw";
  string t = "";
  for(int _ = 0; _ < 200; _++) {
    t += s;
  }

  for(int i = 0; i < t.size() - (w + b); i++) {
    int cw = 0;
    int cb = 0;

    for(int j = 0; j < w + b; j++) {
      if(t[i + j] == 'w') {
        cw++;
      } else {
        cb++;
      }
    }

    if(cw == w && cb == b) {
      cout << "Yes";
      return 0;
    }
  }

  cout << "No";
}

File: abc/abc346/abc346_b/abc346_b.py
w, b = map(int, input().split())
s = 'wbwbwwbwbwbw' * 200

for i in range(len(s) - (w + b)):
  cw = 0
  cb = 0
  for j in range(w + b):
    if s[i + j] == 'w':
      cw += 1
    else:
      cb += 1
  
  if cw == w and cb == b:
    print('Yes')
    exit()

print('No')